concrete_sigar_test.go

```diff
@@ -32,7 +35,9 @@ var _ = Describe("ConcreteSigar", func() {
 			firstValue := <-samplesCh
 
 			secondValue := <-samplesCh
-			Expect(secondValue.User).To(BeNumerically("<", firstValue.User))
+			if runtime.GOOS != "darwin" {
+				Expect(secondValue.User).To(BeNumerically("<", firstValue.User))
+			}
```

Comment:
Good idea - this way the test will fail when we implement the CPU collection on darwin.